web/src/components/core/FileViewer.test.jsx

```diff
@@ -28,19 +28,19 @@ import cockpit from "../../lib/cockpit";
 
 jest.mock("../../lib/cockpit");
 
-const readFn = jest.fn((arg) => new Promise(jest.fn()));
+const readFn = jest.fn();
 
 const fileFn = jest.fn();
 fileFn.mockImplementation(() => {
   return {
     read: readFn
-  }
+  };
 });
 
 cockpit.file.mockImplementation(fileFn);
 
 // testing data
-const file_name = "/testfile"
+const file_name = "/testfile";
```

Comment:
Actually not related with the issue discussed here, but _needed_ to make ESLint happy.

Reply:
I had the same problem in #414. After all pending PRs are merged, we should do something about that.